src/DocumentView/DocumentView.tsx

```diff
@@ -268,7 +269,9 @@ export class DocumentView extends PureComponent<DocumentViewProps, any> {
     } else if (event.nativeEvent.onDocumentError) {
       if (this.props.onDocumentError) {
         this.props.onDocumentError(event.nativeEvent.onDocumentError);
-      } else {
+      } else if (this.props.onError) {
```

Comment:
logic is incorrect in this block, should be more like 

if (this.props.onDocumentError || this.props.onError) {
} else {
}

you should read your changes and look at whether it's what you want, also you should always properly format your code....